robot/impl/local_robot.go

```diff
@@ -1183,6 +1184,37 @@ func (r *localRobot) reconfigure(ctx context.Context, newConfig *config.Config,
 
 	var allErrs error
 
+	// Maintenance config can be configured to block reconfigure based off of a sensor reading
+	// These sensors can be configured on the main robot, a fragment, or a remote
+	// In situations where there are conflicting sensor names the following behavior happens
+	// Main robot and remote share sensor name -> main robot sensor is chosen
+	// Main robot and fragment share sensor name -> main robot sensor is chosen
+	// Only remote has the sensor name -> remote sensor is read
+	// Multiple remotes share a senor name -> conflict error is returned and reconfigure happens
+	// To specify a specific remote sensor use the name format remoteName:sensorName to specify a remote sensor
+	if newConfig.MaintenanceConfig != nil {
+		name, err := resource.NewFromString(newConfig.MaintenanceConfig.SensorName)
+		if err != nil {
+			r.logger.Infof("Sensor Name %s is not in a supported format", newConfig.MaintenanceConfig.SensorName)
```

Comment:
Warn or error log?

Reply:
swapping this and the others over to warn since RDK and reconfigure will still work properly